src/components/LogoutLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'react-router/lib/Link';
+
+/**
+ * Logout link component
+ * @module LogoutLink
+ */
+class LogoutLink extends React.Component {
+	render() {
+		const {
+			to,
+			children,
+			...other
+		} = this.props;
+
+		return (
+			<Link
+				to={{ pathname: to, query: {logout: true} }}
```

Comment:
it's pretty straightforward to manipulate history state - [docs](https://github.com/ReactTraining/react-router/blob/master/docs/guides/NavigatingOutsideOfComponents.md#navigating-outside-of-components) - but my initial preference would be to keep the `logout` query param as a step in the navigation history because it's a pretty big change to application state, and distinct from the view/URL that comes before it, so it might be a little strong to `replaceState`